Task/MCU/Floor1_USART_inputdata/Core/Src/main.c

```diff
@@ -252,6 +304,52 @@ void SystemClock_Config(void)
   }
 }
 
+/**
+  * @brief IWDG Initialization Function
+  * @param None
+  * @retval None
+  */
+static void MX_IWDG_Init(void)
+{
+
+  /* USER CODE BEGIN IWDG_Init 0 */
+#ifdef I_W_D_G
+	prescaler = (CLOCK_FREQUENCY * IWDG_TIMEOUT) / RELOAD_RANGE;
+	uint8_t prescalerIndex;  // Default index
+	// Perform bitwise right shift operations to determine the index
+	prescalerIndex = (prescaler <= 4) ? 0 :
+	                 (prescaler <= 8) ? 1 :
+	                 (prescaler <= 16) ? 2 :
+	                 (prescaler <= 32) ? 3 :
+	                 (prescaler <= 64) ? 4 :
+	                 (prescaler <= 128) ? 5 : 6;
+	uint32_t prescalerValue = 1 << prescalerIndex;
```

Comment:
Why do we need to create separate variable for  prescalerValue ? Can't we put directly **1 << prescalerIndex** where it uses?

Reply:
we can do